src/NuGetGallery/App_Start/DefaultDependenciesModule.cs

```diff
@@ -1458,29 +1462,21 @@ private static void RegisterAuditingServices(ContainerBuilder builder, IAuditing
                 .SingleInstance();
         }
 
-        private static void RegisterCookieComplianceService(ContainerBuilder builder, ConfigurationService configuration, DiagnosticsService diagnostics)
+        private static void RegisterCookieComplianceService(ConfigurationService configuration, ILoggerFactory loggerFactory)
         {
-            CookieComplianceServiceBase service = null;
+            var logger = loggerFactory.CreateLogger("CookieComplianceService");
```

Comment:
Please use `nameof` if possible so that this string doesn't get stale if we rename the type.

```suggestion
            var logger = loggerFactory.CreateLogger(nameof(CookieComplianceService));
```